app/controllers/user.js

```diff
@@ -1,16 +1,41 @@
 const validations = require('../util');
 const userService = require('../services/users');
 const logger = require('../logger');
+const errors = require('../errors');
 
-exports.createUser = ({ body }, res, next) =>
-  validations
-    .passwordEncryption(body)
+exports.createUser = (req, res, next) => {
+  logger.info('createUser method start.');
+  const user = req.body;
+  return validations
+    .passwordEncryption(user)
     .then(userService.userRegister)
     .then(response => {
-      logger.info(`User ${body.firstName} ${body.lastName} created successfully`);
+      logger.info(`User ${user.firstName} ${user.lastName} created successfully`);
       return res.status(201).send({
         firstName: response.firstName,
         lastName: response.lastName
       });
     })
     .catch(next);
+};
+
+exports.signInUser = (req, res, next) => {
+  logger.info('SignInUser method start.');
+  const { email, password } = req.body;
+  return userService
+    .findUser(email)
+    .then(foundUser => {
```

Comment:
This works but I think that you can improve this. Specially, the part when you return null. What should happen if user does not exist?